tests/bluetooth/bsim_bt/bsim_test_audio/prj.conf

```diff
@@ -166,3 +166,4 @@ CONFIG_BT_CTLR_ADV_ISO_STREAM_COUNT=1
 # Only 1 stream support by controller at this point
 CONFIG_BT_CTLR_SYNC_ISO_STREAM_MAX=1
 CONFIG_BT_CTLR_ISO_TX_BUFFERS=3
+CONFIG_BT_CTLR_CENTRAL_ISO=y
```

Comment:
If we adding this, we could consider removing the guards from the audio tests, but that can be done in a separate PR

Reply:
@mtpr-ot quite possible due to https://github.com/zephyrproject-rtos/zephyr/issues/53446

Last I tried, the data path wasn't working correctly if both BIS and CIS is enabled in the build

Reply:
Enabling both CIS and BIS is supported in https://github.com/zephyrproject-rtos/zephyr/pull/52955, and the said babblesim test has been enabled for Connected ISO, passing in that PR.

(but, I have much more changes in local copy that now supports bidirectional encrypted connected ISO, streaming audio in our nRF5340 audio DK).